include/rtdef.h

```diff
@@ -1041,11 +1041,12 @@ struct rt_device_graphic_info
 {
     rt_uint8_t  pixel_format;                           /**< graphic format */
     rt_uint8_t  bits_per_pixel;                         /**< bits per pixel */
-    rt_uint16_t reserved;                               /**< reserved field */
 
     rt_uint16_t width;                                  /**< width of graphic device */
     rt_uint16_t height;                                 /**< height of graphic device */
 
+    rt_uint32_t reserved;                               /**< reserved field */
```

Comment:
Why move this field to here? And change type to `uint32_t`?

The `rt_uint16_t reserved` is 2bytes width and aligns with `pixel_format` and `bits_per_pixel` as a 4bytes width. You can calculate the width of this structure.